### PR TITLE
Added .travis.yml file for integration with Travis CI

### PR DESCRIPTION
**Pull request by scohar**
Merging into master so that we can use travis CI as soon as possible. The added code is very small and doesn't interact with the agent, so there's little chance of this change adding bugs.

### CODE REVIEW

File: Tools/src/coverage.sh [hunk 1]
```diff
@@ -30,4 +30,4 @@ done
 go tool cover -html="$profile" -o ./bin/coverage.html
 go tool cover -func="$profile"
 
-rm -rf "$work_dir"
\ No newline at end of file
+rm -rf "$work_dir"
```
Comment (shihuazhang) [inline Tools/src/coverage.sh]: seems there is no change in this line

Comment (shihuazhang) [inline Tools/src/coverage.sh]: seems there is no change for this line

Comment (scohar) [inline Tools/src/coverage.sh]: It's the addition of a new line at the end of the file. i can get rid of it if you'd like.

Comment (scohar) [inline Tools/src/coverage.sh]: Fixed the new line and added badge to the readme.

**Pull request merged**